editor/node_3d_editor_gizmos.cpp

```diff
@@ -3613,6 +3624,21 @@ void CollisionShape3DGizmoPlugin::redraw(EditorNode3DGizmo *p_gizmo) {
 		Vector<Vector3> handles;
 		handles.push_back(Vector3(r, 0, 0));
 		p_gizmo->add_handles(handles, handles_material);
+
+		// Visible Faces
+		if (EDITOR_DEF_RST("editors/3d_gizmos/collision_shape/show_faces", true)) {
+			Ref<ArrayMesh> m = sp->get_debug_arraymesh_faces();
+			if (EDITOR_DEF_RST("editors/3d_gizmos/collision_shape/random_color", true)) {
```

Comment:
Should random colors even be kept in the first place? We don't use random colors for raycasts or 2D collision shapes after all.

Reply:
It could be simplified if it was just a default color and custom colors, but depending how random colors are implemented, it could be pretty simple (and it could be for both 2D and 3D eventually).

Quoting the last message from @Cold-0:
> When the "custom color" would be (re)added, when the collision node is
added, we add a random color and if the "use custom color" is check use it.
And maybe a config in the editor to allow the "use custom color" checked on
node creation ?

We could even make custom color as simple as for Raycast3D and make the value black by default, which means it uses the editor/runtime default.
Then we could have editor settings to generate random custom colors on 2D/3D collision shape creation.

@Calinou Do you think one of these solutions could be ok, since only editor code would be affected? Some users might find it practical for debugging.